src/shogun2-webapp-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/log4j.properties

```diff
@@ -6,7 +6,7 @@ ${symbol_pound} LOGGING PROPERTIES
 ${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}${symbol_pound}
 
 ${symbol_pound} Some example settings
-log4j.rootLogger=DEBUG, stdout
+log4j.rootLogger=TRACE, stdout
```

Comment:
@marcjansen yes. to demonstrate the work of the custom permission evaluator, which logs on trace (if you are NOT admin) :wink: